src/roof_ss.cc

```diff
@@ -0,0 +1,156 @@
+// This file is a part of openscad. Everything implied is implied.
+// Author: Alexey Korepanov <kaikaikai@yandex.ru>
```

Comment:
Just to clarify, most OpenSCAD code is GPL2+. You've seen that and your code contributions will be covered by that license too?

Also for completeness, the application, once built with CGAL is actually GPL3+ due to CGAL using that license and the combination of GPL2+ and GPL3+ results in the final deliverable be GPL3+.